tools/restatectl/src/commands/partition/list.rs

```diff
@@ -103,6 +103,7 @@ pub async fn list_partitions(
     let mut max_epoch_per_partition: HashMap<u32, u64> = HashMap::new();
     for (node_id, node_state) in cluster_state.nodes {
         match node_state.state.expect("node state is set") {
+            node_state::State::Suspect(_) => {}
```

Comment:
My thought was that this is a very transit state and the node will quickly converge to 'dead' or 'alive' so I thought it wasn't necessary to show it at all.

Note that this will depend on which admin node you are talking to. If your admin is only just coming back there is a chance to see nodes (briefly) in Suspect state.

This is why i decided against showing it as 'dead'. But we also can't show it as alive because we don't have any information about what partitions its running.

Reply:
> the node will quickly converge to 'dead' or 'alive' so I thought it wasn't necessary to show it at all. 
Famous last words :)

Reply:
It only can get stuck if the peer failed to update metadata and stayed at an older version for too long. This definitely can happen. Maybe list the node as `dead(supect)`

Reply:
Now it's shows out as follows

```
☠️ Dead nodes
 NODE  LAST-SEEN
 N2    -

🕵 Suspect nodes
 NODE  LAST-ATTEMPTS
 N1    1 second and 406 ms ago
 N4    1 second and 405 ms ago
```